src/pages/home/report/ReportActionCompose/ReportActionCompose.tsx

```diff
@@ -564,22 +557,18 @@ function ReportActionCompose({
                         {hasExceededMaxCommentLength && <ExceededCommentLength />}
                     </View>
                 </OfflineWithFeedback>
+                {isUsingImportedState && !isSmallScreenWidth && (
```

Comment:
```suggestion
                {!isSmallScreenWidth && (
```

Reply:
isUsingImportedState condition already implemented in ImportedStateIndicator Component, right?